.travis.yml

```diff
@@ -45,6 +45,8 @@ matrix:
     script:
 # FIXME: we should fix centos-6 against AWS and then enable it here
     - if [ "$TRAVIS_SECURE_ENV_VARS" = "true" ]; then bundle exec kitchen test ubuntu; fi
+    after_failure:
+    - cat .kitchen/logs/kitchen.log
```

Comment:
There are also instance-specific logs - you'll want to cat those as well.  They're in the same directory, just named `<instance>.log`